packages/http-client-java/emitter/src/type-utils.ts

```diff
@@ -393,3 +398,17 @@ function getDecoratorScopedValue<T>(
   }
   return undefined;
 }
+
+export function scopeContainsJava(scope: string): boolean {
+  return scope
+    .split(",")
+    .map((s) => s.trim())
+    .includes("java");
+}
+
+export function scopeContainsNegativeNonJava(scope: string): boolean {
```

Comment:
better to add some cases in this functions to explain what it covers, the name is a bit tricky

Reply:
Renamed the function to make it easier to understand. Also added docstring on the func.